docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md

```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
     sudo yba-ctl createBackup <output_path> [flags]
     ```
 
-    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Specify the `output_path` where you want the backup `.tar.gz` file to be stored.
+    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Specify the `output_path` where you want the backup `.tgz` file to be stored.
 
-    The following table describes optional flags you can include with the `createBackup` command.
+    The `createBackup` command creates a timestamped `tgz` file for the backup. For example:
 
-    | Flag | Description |
-    | :--- | :---------- |
-    | --data_dir string | Data directory to be backed up. (default: "/opt/yugabyte") |
-    | --exclude_prometheus | Exclude Prometheus metric data from backup. (default: false) |
-    | --exclude_releases | Exclude YugabyteDB releases from backup. (default: false) |
-    | -h, --help | Help for `createBackup`. (default: false) |
-    | --skip_restart | Don't restart processes during execution. (default: false) |
-    | --verbose | Display extra information in the output. (default: false) |
-    | -f, --force (global flag) | Run in non-interactive mode. All user confirmations are skipped. |
-    | --log_level string (global flag) | Log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default: "info") |
+    ```sh
+    $ sudo yba-ctl createBackup ~/test_backup
+    $ ls test_backup/
+    ```
+
+    ```output
+    backup_23-04-25-16-54.tgz  version_metadata_backup.json
```

Comment:
I don't think we want this? Where did version_metadata_backup.json come from that shouldn't be there

Reply:
version_metadata_backup.json comes from here https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#backup-and-restore
If that is incorrect, then we need to change it there as well